fixtures/example-app/android/app/build.gradle

```diff
@@ -76,6 +76,10 @@ project.ext.react = [
     entryFile: "index.js"
 ]
 
+project.ext.react = [
+  cliPath: "node_modules/haul/bin/cli.js"
+]
+
```

Comment:
Can you remove it? It's because tests run `yarn init` :d

Reply:
Sure, sorry for that :D

Reply:
It's my fault, tests should cleanup after themselves